packages/client/src/contents/__fixtures__/commercepages.fixtures.js

```diff
@@ -1,48 +1,27 @@
-import join from 'proper-url-join';
-import moxios from 'moxios';
+import { rest } from 'msw';
 
-/**
- * @typedef {object} RequestParams
- * @property {object} query - Query to request.
- * @property {Array} [response] - Contents payload.
- */
+const path = '/api/content/v1/commercepages';
 
 /**
  * Response payloads.
  */
 export default {
   get: {
     /**
-     * Success moxios request.
+     * Success msw request.
      *
-     * @param {RequestParams} params - Params to moxios request.
+     * @param {Array} response - Commerce Pages payload.
      */
-    success: params => {
-      moxios.stubRequest(
-        join('/api/content/v1/commercepages', {
-          query: params.queryParams,
-        }),
-        {
-          response: params.response,
-          status: 200,
-        },
-      );
-    },
+    success: response =>
+      rest.get(path, async (req, res, ctx) =>
+        res(ctx.status(200), ctx.json({ ...response })),
```

Comment:
I think you don't need to destructure `{ ...response }` you can just pass response as an argument to `ctx.join(...)`:
```js
ctx.join(response)
```

Reply:
Done